src/commands/new.rs

```diff
@@ -60,11 +72,24 @@ pub async fn new(path: Utf8PathBuf, name: Option<String>) -> Result<(), CliError
     info!("Creating new project at {:?}", dir);
 
     #[cfg(feature = "fetch-template")]
-    let template = match fetch_template().await {
-        Ok(bytes) => bytes,
+    let template: Vec<u8> = match fetch_template().await {
+        Ok(bytes) => {
+            if let Some(cache_file) = cached_template_path() {
+                info!("Storing fetched template in cache.");
+                fs::write(cache_file, &bytes).unwrap_or_else(|_| {
+                    info!("Could not cache template.");
```

Comment:
I know that this isn't really your problem since the previous changes had this issue too, but these should be `error!`s

Reply:
Also the `info!` above should probably be a `debug!`

Reply:
@Tropix126 wouldn't `warn!` make more sense compared to `error!`, since the code keeps running fine after?